hack/ci/e2e-conformance.sh

```diff
@@ -361,14 +365,18 @@ EOF
     return 2
   fi
 
+  if [[ ${1:-} == "--clean" ]]; then
```

Comment:
Should this use the ENV var to align with the doc PR? 

Otherwise, might need to do some shifting of args to support both this and the --verbose flag.

Reply:
you mean support `hack/ci/e2e-conformance.sh --clean --verbose` ? can we do that later, merge and iterate

Reply:
Sure, but should the docs be updated for the flag rather than the env var?